managementconsole/__init__.py

```diff
@@ -0,0 +1,38 @@
+import os
```

Comment:
This needs to be removed and replaced with the flask instance that already exists in master